tidy3d/plugins/invdes/optimizer.py

```diff
@@ -65,6 +66,12 @@ class AbstractOptimizer(InvdesBaseModel, abc.ABC):
         "last computed state of these variables.",
     )
 
+    parameter_spec: ParameterSpecType = pd.Field(
+        default_factory=lambda: UniformParameterSpec(value=0.5),
```

Comment:
Why does it need a factory? And not sure how the docs will render this

Reply:
yeah doesn't need it, for some reason I thought it'd be nicer to not instantiate before calling the class, but it seems a bit pointless. removed it